controllers/airline_controller_test.go

```diff
@@ -39,6 +42,34 @@ func TestHandleAirlineByIdController(t *testing.T) {
 	mockService.EXPECT().GetAirlineById(gomock.Any(), ctx, gomock.Any()).Return(&mockAirlines, nil)
 	ctx.Request, _ = http.NewRequest("GET", "airline/12332", nil)
 	controllerRepo.HandleGetAirlineById(ctx)
+	fmt.Println(ctx.Writer.Status())
 	assert.Equal(t, http.StatusOK, ctx.Writer.Status())
 
 }
+
+func TestHandleCreateNewAirline(t *testing.T) {
```

Comment:
Need other scenarios as well - for eg, re-creating with the same name, etc - and all other negative scenarios

Reply:
done